web/rules/scripts/modified-dates.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import subprocess
+from datetime import datetime, timedelta
+
+start_dir = sys.argv[1]
+output_file = sys.argv[2]

```

Comment:
Maybe use argparser with descriptions for these dirs

Reply:
@DeeyaSingh

Reply:
That's a great idea! I'll work on changing that